include/base/FenixApp.h

```diff
@@ -1,11 +1,16 @@
-//* This file is part of the MOOSE framework
-//* https://www.mooseframework.org
+//* This file is part of FENIX: Fusion ENergy Integrated multiphys-X,
+//* A multiphysics application for modeling plasma facing components
+//* https://github.com/idaholab/fenix
```

Comment:
```suggestion
//* https://github.com/idaholab/fenix
//* https://mooseframework.inl.gov/fenix
```
I would consider also adding the URL to what-will-be the open FENIX website. Commenting once since you have a script to update the rest. This is optional, since the website isn't live yet and we could push another PR.

Reply:
I'll add it to all the headers now

Reply:
Done